internal/pipeline.go

```diff
@@ -70,15 +71,17 @@ func NewPipeline(components []string, isParent bool) (*Pipeline, error) {
 				return nil, err
 			}
 
-			if !(comp.Priority() <= lastPriority) {
-				log.Err("Pipeline::NewPipeline : Invalid Component order [priority of %s higher than above components]", comp.Name())
-				return nil, fmt.Errorf("config error in Pipeline [component %s is out of order]", name)
-			} else {
-				lastPriority = comp.Priority()
-			}
+			if !common.GenConfig {
```

Comment:
Do we have to add some condition exclusion in unit test file for this.?

Reply:
Added test for this